kafka-connect-temp/src/main/scala/com/datamountaineer/streamreactor/temp/ConsistencyLevelSettings.scala

```diff
@@ -0,0 +1,24 @@
+package com.datamountaineer.streamreactor.temp
+
+import org.apache.kafka.common.config.ConfigException
+
+import scala.util.{Failure, Success, Try}
+
+trait ConsistencyLevelSettings extends BaseSettings {
+  val consistencyLevelConstant: String
+
+  def getConsistencyLevel[T <: Enum[T]](clazz: Class[T]): Option[T] = {
+
+    val consistencyLevel = getString(consistencyLevelConstant) match {
+      case "" => None
+      case other =>
+        Try(Enum.valueOf[T](clazz.asInstanceOf[Class[T]], other)) match {
+          case Failure(e) => throw new ConfigException(s"'$other' is not a valid $consistencyLevelConstant. " +
```

Comment:
It's just two lines. Doing multiline -> stripmargin -> replace("\n","") seems a bit less legible than just continuing on the next line.